### ISSUE TITLE
CASIA, cat+tiger, and Cars Pre-trained Models and Code

### ISSUE THREAD

**Issue by ngoc-nguyen-0** [njuaplusplus/mirror#2]
Hi,

Thank you for your sharing!

Can you also share the pre-trained models from other datasets (CASIA, cat+tiger, Cars)? If it is possible, could you share with us the code to train the target/evaluation models?

Thank you!
Ngoc

**Comment by njuaplusplus** [comment]
Hi Ngoc,

All of the pre-trained face models we used are downloaded from existing repositories. Please refer to [this section in readme](https://github.com/njuaplusplus/mirror#acknowledgement).

For the cats, tigers, or cars, we just trained ResNet18 or ResNet34 on them. You can take a look at [this example](https://www.kaggle.com/deepbear/pytorch-car-classifier-90-accuracy/comments).

Best,
Shengwei
